### PR TITLE
only send sparse representation to graph

### PR DESCRIPTION
**Pull request by yingjieMiao**
@leogrim

### CODE REVIEW

File: kifi-backend/modules/cortex/app/com/keepit/common/commanders/FeatureSQSQueueCommander.scala [hunk 1]
```diff
@@ -28,10 +28,16 @@ class FeatureSQSQueueCommander(
...
       val vseq = CortexVersionedSequenceNumber[NormalizedURI](version.version, uri.seq.value)
-      LDAURITopicGraphUpdate(uri.id.get, vseq, "dense_lda", feat.vectorize)
+      val (topicIds, topicScores) = generateSparseRepresentation(feat.vectorize)
+      LDAURITopicGraphUpdate(uri.id.get, vseq, "dense_lda", feat.vectorize.length, topicIds, topicScores)
     }
   }
 
+  private def generateSparseRepresentation(topicVector: Array[Float]): (Array[Int], Array[Float]) = {
+    val s = topicVector.zipWithIndex.sortBy(-1f * _._1).take(5).map{case (score, idx) => (idx, score)}
+    (s.map{_._1}, s.map{_._2})
```
Comment (leogrim) [inline kifi-backend/modules/cortex/app/com/keepit/common/commanders/FeatureSQSQueueCommander.scala]: "unzip" maybe?

Comment (yingjieMiao) [inline kifi-backend/modules/cortex/app/com/keepit/common/commanders/FeatureSQSQueueCommander.scala]: cool. didn't know that.

File: kifi-backend/modules/common/app/com/keepit/graph/manager/GraphUpdate.scala [hunk 2]
```diff
@@ -122,7 +122,9 @@ case class LDAURITopicGraphUpdate(
   uriId: Id[NormalizedURI],
   uriSeq: CortexVersionedSequenceNumber[NormalizedURI],
   modelName: String,
-  topics: Array[Float]
+  dimension: Int,
+  topicIds: Array[Int],
+  topicScores: Array[Float]
```
Comment (leogrim) [inline kifi-backend/modules/common/app/com/keepit/graph/manager/GraphUpdate.scala]: Can you use a Map (higher level class)?

Comment (yingjieMiao) [inline kifi-backend/modules/common/app/com/keepit/graph/manager/GraphUpdate.scala]: As discussed, serialize map requires string as key. not very fun to cast back and forth. IMHO, this interface isn't difficult to interact with.

Comment (leogrim) [inline kifi-backend/modules/common/app/com/keepit/graph/manager/GraphUpdate.scala]: You can copy the code I used to serialize SimpleGraph. It's better to define APIs with higher level classes than specific implementations.

**Comment by yingjieMiao** [comment]
alright, i will just define a case class wrapper anyway

**Comment by yingjieMiao** [comment]
@leogrim 

added a case class

**Comment by leogrim** [comment]
:+1:

**Pull request merged**